### ISSUE TITLE
COLOR_IMAGE_PATH was not used in your code

### ISSUE THREAD

**Issue by hliu869** [yuki-inaho/boundary_loss_np#1]
hello:
when i want to run this code in my PC, i found that 'COLOR_IMAGE_PATH' in your code was not used in the code to calculate the boundary loss, is there any problems in your code when get the boundary loss result?
best regard.

**Comment by yuki-inaho** [comment]
Hello ShyHG,

I just want to confirm about your question. Is it correct that your question is whether the image read via cv2.imread(COLOR_IMAGE_PATH) affects the loss value or not ?
In the sense of the loss value, the color image does not affect anything about it. Thank you.

**Comment by hliu869** [comment]
Hello, yuki:
yes, that is what i want to ask you. but i don't understand what ' the color image does not affect anything about it' means, i think the loss was affected by the prediction and the dist map, and the prediction was calculated via a model like: prediction = model(data), the data was from color image.  
But in your code, loss was affectd by 'one hot array' and 'dist_image', and  'dist_map' was got by : dist_image = one_hot2dist(one_hot_array, n_classes), the 'dist_image' and 'one hot array' was relevant and can they get the right result?
Thank you.

**Comment by yuki-inaho** [comment]
Hello ShyHG,

> yes, that is what i want to ask you. but i don't understand what ' the color image does not affect anything about it' means, i think the loss was affected by the prediction and the dist map, and the prediction was calculated via a model like: prediction = model(data), the data was from color image.

What you said is correct. In actual situation, a category map is generated from a color image via segmentation models.
So, the color images affect the loss clearly in the term of relevance.

But the situation I thought on my previous comment is just in the case in boundary_loss_np.ipynb.
And in the case of the code, the color image is only used for color&label pair visualization (not contributing post-processes).
Sorry for your confusing.

> But in your code, loss was affectd by 'one hot array' and 'dist_image', and 'dist_map' was got by : dist_image = one_hot2dist(one_hot_array, n_classes), the 'dist_image' and 'one hot array' was relevant and can they get the right result?

I have never confirmed the correctness of the loss I implemented and the behavior of my code well,
and I have never used this code to actual learning processes so far. 
The purpose of the code is just for my intuitive confirmation with the toy example.

But, when I implemented my code, I used below official code as reference. Thank you.
https://github.com/LIVIAETS/boundary-loss/blob/749602d822fcb097c79a7fb708805b8982c37030/losses.py#L76-L93
https://github.com/LIVIAETS/boundary-loss/blob/749602d822fcb097c79a7fb708805b8982c37030/dataloader.py
